packages/styles/src/placeholders/_color.scss

```diff
@@ -5,42 +5,56 @@
 
 // background rgb properties
 %color-background-rgb {
-  background-color: rgb(var(--bg-rgb)) !important;
-  color: var(--contrast-color) !important;
+  background-color: rgb(var(--post-bg-rgb)) !important;
+  color: var(--post-contrast-color) !important;
 }
 
 // background rgba properties
 %color-background-rgba {
-  --bg-opacity: 1;
-  background-color: rgba(var(--bg-rgb), var(--bg-opacity)) !important;
-  color: var(--contrast-color) !important;
+  --post-bg-opacity: 1;
+  background-color: rgba(var(--post-bg-rgb), var(--post-bg-opacity)) !important;
+  color: var(--post-contrast-color) !important;
 }
 
 // light background variables
 %color-background-light-variables {
-  --contrast-color: #{color.$black};
-  --contrast-color-rgb: #{color-fn.rgb-values(color.$black)};
-  --contrast-color-inverted: #{color.$white};
-  --contrast-color-inverted-rgb: #{color-fn.rgb-values(color.$white)};
+  --post-contrast-color: #{color.$black};
+  --post-contrast-color-inverted: #{color.$white};
 
-  @each $color-name, $color-value in color.$grays {
-    --gray-#{$color-name}: #{$color-value};
+  @each $color-name, $color-value in color.$post-grays {
+    --post-gray-#{$color-name}: #{$color-value};
+  }
+
+  @each $name, $color in color.$background-colors {
+    --post-#{$name}: #{$color};
+  }
+
+  // RGB variants of the above, moved down here for better overview in the dev tools
+  --post-contrast-color-rgb: #{color-fn.rgb-values(color.$black)};
+  --post-contrast-color-inverted-rgb: #{color-fn.rgb-values(color.$white)};
+
+  @each $color-name, $color-value in color.$post-grays {
+    --post-gray-#{$color-name}-rgb: #{color-fn.rgb-values($color-value)};
+  }
+
+  @each $name, $color in color.$background-colors {
+    --post-#{$name}-rgb: #{color-fn.rgb-values($color)};
   }
 }
 
 // dark background variables
 %color-background-dark-variables {
```

Comment:
The light theme will be the default for the time being. A dark theme could be introduced by a class on the body or by using the users preferred theme. Dark theme styles will override the default light ones. If that is going to change in the future, we'll need to update this in the future. Doing so now might introduce unnecessary complexity.

When the dark theme is designed, it may be possible that some accent colors need adjustments for the dark mode.